src/renderRepoCard.js

```diff
@@ -49,6 +50,14 @@ const renderRepoCard = (repo, options = {}) => {
   const totalStars = kFormatter(stargazers.totalCount);
   const totalForks = kFormatter(forkCount);
 
+  const templateBadge = `
+    <g data-testid="template-badge" transform="translate(320, 24)">
+      <rect rx="10" ry="20" width="60" height="20" fill="rgb(255, 255, 255, 0)" stroke="#e1e4e8" >
+      </rect>
+    <text x="30" y="11" alignment-baseline="middle" text-anchor="middle" font-weight="600" font-size="12" fill="#586069" font-family="400 13px 'Segoe UI', Ubuntu, Sans-Serif">Template</text>
+    </g>
```

Comment:
use the correct color values from textColor instead of hard coding them. just copy paste the archiveBadge template and change the dimensions as necessary

Reply:
Fixed color on this commit https://github.com/anuraghazra/github-readme-stats/pull/146/commits/6b23ed79552682a400d7da39587dfafd45d3eed1

Reply:
`textColor` changed the looks from the original

**github**
![image](https://user-images.githubusercontent.com/15318294/88208865-7c799380-cc6f-11ea-818c-830a28c77c5b.png)
**github-readme-stats**
![image](https://user-images.githubusercontent.com/15318294/88208931-90bd9080-cc6f-11ea-886e-71ba9abbc59a.png)

Reply:
It is the same, check out the code of archive i have `.archive-badge rect { opacity: 0.2 }` setup which gives good results.

Reply:
now it's looks good